conda_build_config.yaml

```diff
@@ -1,11 +1,8 @@
 python:
-  - 3.7.3
+  - 3.7
 
-liblapack:
-  - 3.8.0
-
-_openmp_mutex:
-  - 4.5 1_gnu
-  
 boost:
   - 1.75.* # see https://github.com/scipp/scipp/issues/1888
```

Comment:
In scipp we are pinning to `1.76` i think... this seems inconsistent?

Reply:
scipp only uses boost as a build dependency. For mantid it's both a build and run dependency. I believe that 1.75 is as far as Mantid is compatible with boost. I'll double check the native mantid conda recipe though.